Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -63,6 +63,7 @@ extension NavigationMapView {
         guard !routeProgress.routeIsComplete else {
             removeRoutes()
             removeContinuousAlternativesRoutes()
+            removeArrow()
```

Comment:
This issue only happens when vanishing route line functionality is enabled?

Reply:
Yes. If the vanishing route line functionality is not enabled, the route line will not be removed. So it's reasonable to keep the arrow. 
And we only remove the route and route line when the vanishing route line functionality is enabled and route is complete. So it's safe that we check whether the route is removed after  `navigationMapView?.updateRouteLine(routeProgress:coordinate:shouldRedraw:)`, and if the route is removed, we should remove the arrow.